v2/internal/controllers/crd_servicebus_namespace_standard_test.go

```diff
@@ -97,4 +103,50 @@ func ServiceBus_Topic_CRUD(tc *testcommon.KubePerTestContext, sbNamespace client
 	// a basic assertion on a property
 	tc.Expect(topic.Status.SizeInBytes).ToNot(BeNil())
 	tc.Expect(*topic.Status.SizeInBytes).To(Equal(0))
+
+	tc.RunParallelSubtests(
```

Comment:
I dont think this should be in the `Secrets` test. Maybe just as a subtest of the "normal" test?

Also the name you are calling it here is "Namespace secrets", which doesn't seem right. Call it `Subscription CRUD` maybe?